.github/workflows/playground-check.yaml

```diff
@@ -0,0 +1,68 @@
+name: Playground operation check
+
+on:
+  pull_request:
+  schedule:
+    # Run this job once per day
+    - cron: "0 0 * * *"
+
+jobs:
+  playground-check:
+    runs-on: ubuntu-latest
+    env:
+      FORK_URL_MAINNET: ${{ secrets.FORK_URL_MAINNET }}
+    steps:
+      - name: Free Disk Space (Ubuntu)
+        uses: jlumbroso/free-disk-space@main
+        with:
+          tool-cache: false
+          android: true
+          dotnet: true
+          haskell: true
+          large-packages: false
+          docker-images: false
+          swap-storage: false
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install dependency
+        run: sudo apt-get -qq update && sudo apt-get -y -q install curl jq
+
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+
+      - name: Setup environment
+        run: |
+          cp playground/.env.example playground/.env
+          sed -i -e 's/ETH_RPC_URL=.*/ETH_RPC_URL=$FORK_URL_MAINNET/g' playground/.env
+          cat playground/.env
+
+      - name: Start docker containers
+        run: |
+          cd playground
+          # firstly start explorer container to workaround yarn cache problem 
```

Comment:
After resolving free space issue, starting containers were failing with wired error like this:
```
error An unexpected error occurred: "https://registry.yarnpkg.com/through/-/through-2.3.8.tgz: ENOENT: no such file or directory, open '/home/node/.yarn/v6/npm-through-2.3.8-0dd4c9ffaabc357960b1b724115d7e0e86a2e1f5-integrity/node_modules/through/.yarn-tarball.tgz'".
62.73 info If you think this is a bug, please open a bug report with the information provided in "/usr/src/app/yarn-error.log".
62.73 info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```
or
```
20.35 error https://registry.yarnpkg.com/date-fns/-/date-fns-2.30.0.tgz: Extracting tar content of undefined failed, the file appears to be corrupt: "EEXIST: file already exists, mkdir '/home/node/.yarn/v6/npm-date-fns-2.30.0-f367e644839ff57894ec6ac480de40cae4b0f4d0-integrity/node_modules/date-fns/esm/fp/startOfISOWeekYear'"
```
All runs between [#2](https://github.com/cowprotocol/services/actions/runs/11469152331/job/31915788184)
and [#9](https://github.com/cowprotocol/services/actions/runs/11517393034/job/32061983512) failed because of some `yarn` package issue (there were various files listed).
I assumed this was caused somehow by concurrent `explorer` and `fronted` containers build, they have similar configuration of `yarn` cache in installation step:
`--mount=type=cache,target=/home/node/.yarn YARN_CACHE_FOLDER=/home/node/.yarn`.
This was just a hypothesis I didn't analyzed it further as building all containers sequentially or building at least these both containers sequentially was enough to resolve that issue.
I've found some similar issues one the Internet, for instance: https://github.com/yarnpkg/yarn/issues/8686, but without any special conclusions, it can be also some network related thing.